.github/workflows/review-release-notes.yml

```diff
@@ -1,6 +1,9 @@
 name: Review Release Notes
 on: pull_request
 
+env:
+  CHANGED_FILES_DELIMITER: ","
```

Comment:
No specific reason, we can change it back to semicolon if needed. I reviewed the PR/issue but couldn't find a specific comment about a comma (only single quotes). In any case, both `,` and `;` are [valid in filenames](https://www.ibm.com/docs/en/zos/2.1.0?topic=pages-zos-data-set-unix-file-naming-conventions) so it shouldn't matter  much which delimiter is used. 

In any case, since we're adding the following to the job:

```yaml
          files: |
            Packs/**/ReleaseNotes/**
```

We're only going to get a list back of release notes standard format `$MAJOR_MINOR_REVISION.md` so no commas or semicolons should appear in the filenames we're going to process through the doc review.